src/fuel-vm/instruction-set.md

```diff
@@ -2352,6 +2353,22 @@ Panic if:
 
 All these instructions advance the program counter `$pc` by `4` after performing their operation.
 
+### ECAL: Call external function
```

Comment:
Why not just use ecall?

```suggestion
### ECALL: Call external function
```

Reply:
There's been a convention of having max four characters per opcode. I'm usually trying to hold the conventions if I don't have any special reason to break them. If we do this, maybe we should also rename `ALOC` &rarr; `ALLOC`, `CROO` &rarr; `CROOT`, `CSIZ` &rarr; `CSIZE`? I'm also fine having this be a special case, as I don't think the naming matters that much.

Reply:
ah good point, probably not worth changing the convention